### PR TITLE
adding post about linkdrop

### PR DESCRIPTION
**Pull request by julien51**
# Description

All in the title... this week's launch!

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

File: unlock-protocol.com/blog/linkdrop.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Linkdrop: granting keys through links!"
+authorName: Julien Genestoux
+publishDate: October 9, 2019
+description: Linkdrop provides a simple mechanism which lets anyone create links to "claim" keys for free!
+image: /static/images/blog/linkdrop/linkdrop.png
+---
+
+In the blockchain world, one of the most potent distribution mechanism is called **an airdrop**. It is a way for token creators to send their tokens to a large set of people, without requiring these recipients to perform transactions and pay fees themselves. Unfortunately it is often used as a vector for spam even though it is a very useful tool to boostrap a fairly wide adoption.
```
Comment (cnasc) [inline unlock-protocol.com/blog/linkdrop.md]:
```suggestion
In the blockchain world, one of the most potent distribution mechanisms is called **an airdrop**. It is a way for token creators to send their tokens to a large set of people, without requiring these recipients to perform transactions and pay fees themselves. Unfortunately it is often used as a vector for spam even though it is a very useful tool to boostrap a fairly wide adoption.
```

File: unlock-protocol.com/blog/linkdrop.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+At Unlock, [we believe](/blog/mission-vision/) that all creators should make a sustainable living from their work, while respecting the time, privacy and health of their community... and spam does not really fit in this vision!
+
+Additionally, when talking with creators, they also often mention that they want to give away some access keys to their most loyal fans or others members of their community as a way to thank them for their contributions... Of course, our smart contract already includes a `purchaseFor` method which anyone can invoke in order to purchase a new key and assign it to someone else. In the Ethereum world, this is akin to a "meta transaction", a transaction whose beneficary is not its sender.
```
Comment (cnasc) [inline unlock-protocol.com/blog/linkdrop.md]:
```suggestion
Additionally, when talking with creators, they often mention that they want to give away some access keys to their most loyal fans or others members of their community as a way to thank them for their contributions. Of course, our smart contract already includes a `purchaseFor` method which anyone can invoke in order to purchase a new key and assign it to someone else. In the Ethereum world, this is akin to a "meta transaction", a transaction whose beneficiary is not its sender.
```

File: unlock-protocol.com/blog/linkdrop.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+Unfortunately, this function, `purchaseFor` requires to know the recipient's Ethereum address in advance which is not always practical. Luckily, the team at [LinkDrop](https://linkdrop.io/) found a very good way to solve this: they created a system which lets anyone _create links_ with attached keys. The first user who clicks on a link can then claim a unique key for it. This way, someone who wants to give away keys to a lock can easily do so by creating droplinks which they would then send to users by email, social networking, or even for promotional events.
```
Comment (cnasc) [inline unlock-protocol.com/blog/linkdrop.md]:
```suggestion
Unfortunately, this function, `purchaseFor` requires the purchaser to know the recipient's Ethereum address in advance which is not always practical. Luckily, the team at [LinkDrop](https://linkdrop.io/) found a very good way to solve this: they created a system which lets anyone _create links_ with attached keys. The first user who clicks on a link can then claim a unique key for it. This way, someone who wants to give away keys to a lock can easily do so by creating droplinks which they would then send to users by email, social networking, or even for promotional events.
```

Comment (artignatyev) [reply unlock-protocol.com/blog/linkdrop.md]:
```suggestion
Unfortunately, this function, `purchaseFor` requires to know the recipient's Ethereum address in advance which is not always practical. Luckily, the team at [Linkdrop](https://linkdrop.io/) found a very good way to solve this: they created a system which lets anyone _create links_ with attached keys. The first user who clicks on a link can then claim a unique key for it. This way, someone who wants to give away keys to a lock can easily do so by creating droplinks which they would then send to users by email, social networking, or even for promotional events.
```
Linkdrop without capital "D", thanks

**Comment by julien51** [comment]
Thanks a LOT!

**Pull request merged**